javascript/.eslintrc.json

```diff
@@ -0,0 +1,23 @@
+{
```

Comment:
The indentation looks off in this file. Mind `COMMAND+OPTION+L`'ing this file to get it formatted?

Reply:
fixed: https://github.com/Invoca/style-guide/pull/7/commits/447a83b069b8eb932bbce25702830320e16a084e